### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by sorrymaker1130**

### CODE REVIEW

File: lab-exercise-1/WYFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class WYFeature {
+    public void feature() {
+        // this is a new feature
```
Comment (TreephobKho) [inline lab-exercise-1/WYFeature.java:3]: Please include this part.

Comment (Pakin-aw) [inline lab-exercise-1/WYFeature.java:3]: Nice comment.

Comment (ACitronella) [inline lab-exercise-1/WYFeature.java:3]: nice feature

File: lab-exercise-1/WYFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+public class WYFeature {
```
Comment (BunradarCH) [inline lab-exercise-1/WYFeature.java:1]: good feature

File: lab-exercise-1/WYFeature.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+    public void feature() {
+        // this is a new feature
+    }
```
Comment (sorrymaker1130) [inline lab-exercise-1/WYFeature.java:4]: Please add more function here.

**Comment by sorrymaker1130** [comment]
The new fix is avaliable now.

File: lab-exercise-1/WYFeature.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+public class WYFeature {
+    public void feature() {
+        // this is a new feature
```
Comment (RolinAnn) [inline lab-exercise-1/WYFeature.java:3]: Thank u for help me.

File: lab-exercise-1/WYFeature.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+        int x = 0;
+        x = x+1;
```
Comment (sorrymaker1130) [inline lab-exercise-1/WYFeature.java:5]: Add the part successful.

File: lab-exercise-1/WYFeature.java [hunk 6]
```diff
@@ -0,0 +1,7 @@
+public class WYFeature {
+    public void feature() {
+        // this is a new feature
+        int x = 0;
```
Comment (Pakin-aw) [inline lab-exercise-1/WYFeature.java:4]: Nice

Comment (RolinAnn) [inline lab-exercise-1/WYFeature.java:4]: Thank you for adding this part.

File: lab-exercise-1/WYFeature.java (hunk 5 above)
Comment (ACitronella) [inline lab-exercise-1/WYFeature.java:5]: outstanding change

File: lab-exercise-1/WYFeature.java [hunk 7]
```diff
@@ -0,0 +1,7 @@
+public class WYFeature {
```
Comment (BunradarCH) [inline lab-exercise-1/WYFeature.java:1]: good

**Pull request merged**